src/components/generics/vacation-test-forms/employee-vacation-requests.tsx

```diff
@@ -68,17 +72,26 @@ const RenderEmployeeVacationRequests = ({ persons }: Props) => {
   const { accessToken } = useAppSelector(selectAuth);
   const context = useContext(ErrorContext);
   const [ requests, setRequests ] = useState<VacationRequest[]>([]);
+  const [ statuses, setStatuses ] = useState<VacationRequestStatus[]>([]);
+  const [ latestRequestStatuses, setLatestRequestStatuses ] = useState<VacationRequestStatus[]>([]);
   const [ openRows, setOpenRows ] = useState<boolean[]>([]);
   const [sortBy, setSortBy] = useState<VacationRequestSort>(VacationRequestSort.START_DATE);
   const [sortOrder, setSortOrder] = useState<"asc" | "desc">("asc");
+  const [ loading, setLoading ] = useState<boolean>(false);
+  const [ approvalChanged, setApprovalChanged ] = useState<boolean>(false);
+  const myRef = useRef<any>();
 
   /**
    * Initializes all vacation requests
    */
   const initializeRequests = async () => {
     try {
       const vacationsApi = Api.getVacationRequestsApi(accessToken?.access_token);
-      const vacations = await vacationsApi.listVacationRequests({});
+
+      // Hardcoded personId for testing purposes, use person.keyCloakId for staging 
+      const vacations = await vacationsApi.listVacationRequests({ personId: person?.keycloakId });
```

Comment:
Did you mean to remove this line?
const vacations = await vacationsApi.listVacationRequests({ personId: person?.keycloakId });
or should it be like this?
const vacations = await vacationsApi.listVacationRequests({ });

The code would break, I assume the comments (at line 91 and 93) could be removed though?

Reply:
I removed comments around 'const vacation = ...' declaration.